install-proto.sh

```diff
@@ -0,0 +1,41 @@
+mkdir "${GOPATH}"/src/github.com/gogo
+
+cd "${GOPATH}"/src/github.com/gogo
+
+git clone https://github.com/gogo/protobuf.git
+
+cd protobuf
+
+git fetch origin pull/659/head:casttypewith
+
+git checkout casttypewith
+
+PROTO_COMPILER=protoc-3.17.3-linux-x86_64.zip
+TEMP_FOLDER_NAME=temp-001
+
+mkdir ~/${TEMP_FOLDER_NAME}
```

Comment:
`mkdir /tmp/${TEMP_FOLDER_NAME}` instead. Also on L18 & L29. Will this make the script fail if the temp folder exists?

Reply:
refactored.